submodules/network/outputs.tf

```diff
@@ -4,11 +4,11 @@ output "vpc_id" {
 }
 
 output "private_subnets" {
-  description = "List of IDs of private subnets"
-  value       = [for sb in aws_subnet.private : sb.id]
+  description = "List of private subnet ID and AZ"
+  value       = [for cidr, c in local.private_cidrs : { subnet_id = aws_subnet.private[cidr].id, az = c.az }]
 }
 
 output "public_subnets" {
-  description = "List of IDs of public subnets"
-  value       = [for sb in aws_subnet.public : sb.id]
+  description = "List of public subnet ID and AZ"
+  value       = [for cidr, c in local.public_cidrs : { subnet_id = aws_subnet.public[cidr].id, az = c.az }]
```

Comment:
this will break ocp.

Reply:
in what way?

Reply:
https://github.com/dominodatalab/terraform-aws-ocp/blob/main/main.tf#L89
list(string) --> list(object{subnet_id= string,az = string})